packages/vite/src/index.ts

```diff
@@ -62,30 +65,29 @@ export default function kumaUI(): Plugin {
       const result = transform(code, id);
       if (!result?.code) return;
       const css = (result.metadata as unknown as { css: string }).css || "";
-      const cssFilename = path.normalize(
-        `${id.replace(/\.[jt]sx?$/, "")}-${generateHash(css)}.css`,
-      );
-      const cssRelativePath = path
-        .relative(process.cwd(), cssFilename)
-        .replace(/\\/g, path.posix.sep);
-      cssLookup[cssRelativePath] = css;
+
+      const cssTableId = `${Date.now().toString(16)}x${cssTable.length}`;
```

Comment:
@AsPulse 
Thank you for your PR!
It seems that the import URL for the virtual module changes every time we build. Is it possible to hash the relative (or absolute) paths in the CSS to ensure that they remain unchanged even after building?

Reply:
Of course it is possible,
but since, when the component changed, the import path for the virtual module were still changing every time we build.
It is because current behavior adds hashed the "content" of the CSS to path.

If it's not too much trouble, could you let me know why keeping import path is better...?

Reply:
I believe that with your current code, the hash is being changed every time a file is saved, even if the Kuma UI component styles remain unchanged. This leads to a change in the CSS import path each time. I'm commenting with the intention of requesting that the hash be changed only when there is a change in the styles, just like it was before.

By doing this, we can avoid the scenario where CSS is reloaded with HMR every time a file is saved, even when there is no change in the styles. If we only change the hash when there is a change in the styles, HMR should work smoothly.

ref: https://discord.com/channels/1119672933477011598/1127180003872886794/1174362017491058718

Reply:
Additional information: it seems that a full reload is triggered in the Vite plugin's handleHotUpdate, so even when saving a file without any code changes, all CSS and JS are being loaded every time. The handleHotUpdate here is unnecessary, so please remove it.